cannula/models.py

```diff
@@ -885,3 +885,11 @@ def get_validation_view_names():
     cursor = connection.cursor()
     cursor.execute('SELECT viewname FROM pg_catalog.pg_views WHERE viewowner=%s and viewname LIKE %s;', (settings.DATABASES['default']['USER'], 'vw_validation_%'))
     return [x[0] for x in cursor]
+
+class quarterly_reports(models.Model):
```

Comment:
This class is named in a non-standard way, should be 'QuarterlyReport'. But the bigger issue is that it is unnecessary, it seems to only exist to provide a file-upload widget through the admin interface to the media/static directory.